util/global_community_silence.php

```diff
@@ -36,11 +36,11 @@
 use Friendica\Database\DBM;
 use Friendica\Network\Probe;
 
-require_once("boot.php");
+require_once'"boot.php');
```

Comment:
This is invalid. Please remove the double quote and the parenthesis.

Reply:
yea, thick air and I just realized what you wrote ;-)